packages/components/BrandText/BrandTextBase.tsx

```diff
@@ -26,12 +30,17 @@ export const BrandTextBase: React.FC<TextProps & { isTicker?: boolean }> = (
     );
 };
 
-// FIXME: remove StyleSheet.create
-// eslint-disable-next-line no-restricted-syntax
-const styles = StyleSheet.create({
-  base: {
-    color: "white",
-    fontSize: 20,
-    fontWeight: "600",
-  },
-});
+const useStyles = () => {
+  const isLightTheme = useSelector(selectIsLightTheme);
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
This comment is used to prevent an error due to StyleSheet.create, since we didn't yet  replaced all StyleSheet.create. So you can remove it
```suggestion
```

Reply:
Yeah totally true 👍